lib/measured/conversion_table.rb

```diff
@@ -38,20 +42,22 @@ def find_direct_conversion(units, to:, from:)
     nil
   end
 
-  def find_tree_traversal_conversion(units, to:, from:)
-    traverse(units, from: from, to: to, unit_names: units.map(&:name), amount: 1)
+  def find_tree_traversal_conversion(to:, from:)
+    traverse(from: from, to: to, units_remaining: units.map(&:name), amount: 1)
   end
 
-  def traverse(units, from:, to:, unit_names:, amount:)
-    unit_names = unit_names - [from]
+  def traverse(from:, to:, units_remaining:, amount:)
```

Comment:
If I had to nit super hard, I'd keep this one named `unit_names` in the function signature, and call the variable `units_remaining` below, since the "remaining" aspect is internal to the function :)

Reply:
> If I had to nit super hard

Reply:
> If I had to nit super hard

This made me chuckle :joy:

Reply:
Respect. But I changed this because everything in here is names. This is more explicit about what it is. And it's internal. And it's consistent.